node/src/types/max_ttl.rs

```diff
@@ -18,22 +18,18 @@ impl MaxTtl {
     }
 
     /// Determine if two timestamps are more than max_ttl apart.
-    pub fn ttl_elapsed(&self, higher: Timestamp, lower: Timestamp) -> Result<bool, String> {
-        if lower > higher {
-            Err("invalid timestamp chronology".to_string())
-        } else {
-            Ok(lower < higher.saturating_sub(self.0))
-        }
+    pub fn ttl_elapsed(&self, higher: Timestamp, lower: Timestamp) -> bool {
```

Comment:
/// If rearview is earlier than (vantage - ttl duration), ttl has elapsed.
    pub fn ttl_elapsed(&self, vantage: Timestamp, rearview: Timestamp) -> bool {
        rearview < vantage.saturating_sub(self.0)
    }

Reply:
renamed in https://github.com/casper-network/casper-node/pull/4195/commits/ef1b648f1dc0bd9450c9938d77939a6a2a310970